srcs/app/test/integration/delete_method_test.py

```diff
@@ -0,0 +1,103 @@
+import os
+from pathlib import Path
+import platform
+import pytest
+import requests
+import subprocess
+import tempfile
+import time
+
+PROJ_DIR = str(Path(__file__).parents[4])
+TMP_UPLOAD_DIR = PROJ_DIR + '/html/web3/upload/'
+
+@pytest.fixture(scope='module', autouse=True)
+def start_webserv():
+    webserv = subprocess.Popen('./webserv', cwd=PROJ_DIR)
+    time.sleep(2)
+    yield webserv
+    webserv.terminate()
+
```

Comment:
The [pytest documentation](https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session) recommends creating a `conftest.py` file and putting all the fixtures there so that tests from multiple test modules in the directory can access the fixture functions.

This way, we could also add `scope="session"` to some of the fixtures to only be invoked once per test session, and we could import the variables from other modules.
```
from conftest import PROJ_DIR
```
<br>

A useful command to see the setup/teardown of the tests:
```
pytest --setup-show
```

Reply:
Sounds good, though I see this an improvement for another task. If you want to go with it ;) Just a short reminder for this, there is a "problem" with delete tests. Since those are deleting files, you cannot re-use those fixtures and have to create new ones.

Reply:
Ok, I will add a PR for this task after we merge this.